plugins/generator-1.19.2/forge-1.19.2/templates/procedure.java.ftl

```diff
@@ -0,0 +1,94 @@
+ # 
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ # 
+ # Additional permission for code generator templates (*.ftl files)
+ # 
+ # As a special exception, you may create a larger work that contains part or 
+ # all of the MCreator code generator templates (*.ftl files) and distribute 
+ # that work under terms of your choice, so long as that work isn't itself a 
+ # template for code generation. Alternatively, if you modify or redistribute 
+ # the template itself, you may (at your option) remove this special exception, 
+ # which will cause the template and the resulting code generator output files 
+ # to be licensed under the GNU General Public License without this special 
+ # exception.
+-->
+
+<#-- @formatter:off -->
+package ${package}.procedures;
+
+import net.minecraftforge.eventbus.api.Event;
+
+import javax.annotation.Nullable;
+
+<#assign nullableDependencies = []/>
+<#list dependencies as dependency>
+	<#if dependency.getType(generator.getWorkspace()) != "double"
+		&& dependency.getType(generator.getWorkspace()) != "LevelAccessor"
+		&& dependency.getType(generator.getWorkspace()) != "ItemStack"
+		&& dependency.getType(generator.getWorkspace()) != "BlockState"
+		&& dependency.getType(generator.getWorkspace()) != "InteractionResult"
+		&& dependency.getType(generator.getWorkspace()) != "boolean"
+		&& dependency.getType(generator.getWorkspace()) != "CommandContext<CommandSourceStack>">
+		<#assign nullableDependencies += [dependency.getName()]/>
+	</#if>
+</#list>
+
+<#compress>
+
+<#if trigger_code?has_content>
+${trigger_code}
+<#else>
+public class ${name}Procedure {
+</#if>
+
+	<#if trigger_code?has_content>
+	public static <#if return_type??>${return_type.getJavaType(generator.getWorkspace())}<#else>void</#if> execute(
+		<#list dependencies as dependency>
+			${dependency.getType(generator.getWorkspace())} ${dependency.getName()}<#if dependency?has_next>,</#if>
+		</#list>
+	) {
+		<#if return_type??>return </#if>execute(null<#if dependencies?has_content>,</#if><#list dependencies as dependency>${dependency.getName()}<#if dependency?has_next>,</#if></#list>);
+	}
+	</#if>
+
+	<#if trigger_code?has_content>private <#else>public </#if>static <#if return_type??>${return_type.getJavaType(generator.getWorkspace())}<#else>void</#if> execute(
+		<#if trigger_code?has_content>@Nullable Event event<#if dependencies?has_content>,</#if></#if>
+		<#list dependencies as dependency>
+				${dependency.getType(generator.getWorkspace())} ${dependency.getName()}<#if dependency?has_next>,</#if>
+		</#list>
+	) {
+		<#if nullableDependencies?has_content>
+			if(
+			<#list nullableDependencies as dependency>
+			${dependency} == null <#if dependency?has_next>||</#if>
+			</#list>
+			) return <#if return_type??>${return_type.getDefaultValue(generator.getWorkspace())}</#if>;
+		</#if>
+
+		<#list localvariables as var>

```

Comment:
The indentation here is wrong, match indentation as it was in 1.18.2 please

Reply:
It was impossible to see inside Intellij... weird.